about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>About Gamecon</title>
+</head>
+
+<body>
+    <header>
+        <div id="topnav">
+            <a href=""><img src="./images/pngkey.com-facebook-logo-png-white-1532072.png" alt="Facebook"
+                    class="social"></a>
+            <a href=""><img src="./images/pngkey.com-twitter-logo-png-transparent-111779.png" alt="Twitter"
+                    class="social"></a>
```

Comment:
- [Optional] Dear @SnakyMz, good job making the changes suggested by the previous reviewer, Here I can see your navbar is overlapping for a screen size of 1050px. This is not a good practice when it comes to UX and we need to have a responsive design. Please try to work on this.
### Your navbar on 1050px screen.
![image](https://github.com/SnakyMz/Gamecon/assets/69077061/bf367d20-6d1e-44a4-8d58-074f70be0d7d)